tools/async_to_sync.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env python
 """Convert async code in the project to sync code.
 
-Note: the version of Python used to run this script affects the output. Please
-use the `async-to-sync.sh` wrapper to use a version consistent with CI checks.
+Note: the version of Python used to run this script affects the output.
```

Comment:
Oh, thank you for fixing it. I added the build-in-a-container feature as an external script, but I figured out it could have been easy replaced by Python and ended up not committing that version.

Reply:
We can actually follow up on this comment by printing a warning if the Python version used is different from the expected PYVER.

No need to do it in this MR: I can do it in the main async-to-sync branch after merging this MR to avoid conflicts. Unless you jump on it and do it before me.

Reply:
Edit: Never mind about the version check: I've already done it my side.